### ISSUE TITLE
Demo isn't working at the moment

### ISSUE THREAD

**Issue by noatamir** [Aupajo/almanack#35]
Hi, 

I was was looking for a similar solution for combining calendar feeds (but for a privately share link, and not a public calendar), and I noticed your demo link isn't working at the moment.
<img width="641" alt="Screenshot 2019-11-15 at 17 00 20" src="https://user-images.githubusercontent.com/6564007/68957102-a1a53080-07c9-11ea-93c2-8a4f911e3559.png">

Cheers,
Noa

P.S. Is there a way to use your gem without hosting the resulting calendar publicly?

**Comment by Aupajo** [comment]
You're right! It looks like Meetup have just deprecated API keys in favour of OAuth 2, which is why this broke. It might take some time to fix this.

In the meantime, other calendar integrations (Google Calendar, iCal) should work fine.

Yes, you can use the gem without hosting the calendar publicly. How did you want to run it?
